app/views/install/compose.phtml

```diff
@@ -207,6 +208,7 @@ services:
       - _APP_SYSTEM_SECURITY_EMAIL_ADDRESS
       - _APP_REDIS_HOST
       - _APP_REDIS_PORT
+      - _APP_DOMAIN_TARGET
```

Comment:
Why are we using this var in the deletes worker?

Reply:
This got added to the certifiactes worker. It was added to the dev docker-compose last time, but was missing in the production compose.

Reply:
👌🏻